lib/improver/cli/ensemble_calibration.py

```diff
@@ -161,6 +161,29 @@ def main(argv=None):
                              'is converted to percentiles, as part of '
                              'converting the input probabilities into '
                              'realizations.')
+    parser.add_argument('--max_iterations', metavar='MAX_ITERATIONS',
```

Comment:
Hmm, while the CLI is our default interface, I can see an argument for including them at the CLI - do we generally believe that they'll need to be exposed to the end user though? (They seem quite low level). Perhaps we could just figure out appropriate defaults and use those.

For now, I think leaving them here is fine though.

Reply:
Yes, this is a fair point. I think the main usage for this at the moment would be for the case where you're modifying the `predictor_of_mean` argument with the options being `mean` or `realizations` where either the coefficients for the mean or coefficient for the the individual realizations are solved for. There may be justification if solving for coefficients for individual realizations to increase the maximum number of iterations, if there are a number of realizations. As we're considering future experiments involving e.g. modifying the training period, it might be useful to have this option readily available as part of this testing. However, it is fair to say that in the future, we might be happy to just set this variable to a fixed default and sticking with that. For the moment, I've left this option in the CLI.

Reply:
Agreed, happy to leave it also. :)